Y belt attachment with blockers

bstrawsma opened the pull request:
added the blockers integrated into a y belt attachment

bstrawsma committed:
Added blockers

Added a folder in stls for the blockers integrated into the y belt attachment

bstrawsma committed:
Merge branch 'zruncho3d:main' into main

bstrawsma committed:
updated stl

stls should now be in proper order to print as is

bstrawsma committed:
Merge branch 'main' of https://github.com/bstrawsma/double-dragon

bstrawsma committed:
Added new bearing mounts

Added new stls for bearing mounts and blockers integrated into the y belt attachment

bstrawsma committed:
Backwards compatible

Updated the y belt attachments with blockers to be backwards compatible

bstrawsma committed:
Added Nema 17 models

Added modified parts to support Nema 17 motors

zruncho3d commented:
@bstrawsma This is great stuff, and I'd love to test it out and try it.  

Looking at the STL... what orientation did you print it in?  The one for the STL presumes that supports are enabled to cover the thin arm, but I'd probably flip it and deal with some light bridging by the belt attachment.

bstrawsma commented:
@zruncho3d The STL's should be in proper orientation when imported into your slicer. There should be no supports required as per the voron printing specs, but there is some small bridging on the y belt attachments, however even if the bridging isn't perfect it wont effect the function.

bstrawsma committed:
Added V0.1 style nozzle endstop

added sex bolt endstop compatible with v0 bed style for nozzle touch off

bstrawsma committed:
Skr pico Config

Added my config files for running dual skr picos, with still only the standard z leadscrew motor.

bstrawsma commented:
I went ahead and added my config files for wiring up with two skr picos, as well as added my v0.1 style nozzle endstop

bstrawsma committed:
Vernier Calibration

Added vernier calibration print to optional stls
